pkg/server/test/write_authzmodel.go

```diff
@@ -71,18 +88,36 @@ func WriteAuthorizationModelTest(t *testing.T, datastore storage.OpenFGADatastor
 					},
 				},
 			},
+			allowSchema10: false,
```

Comment:
this test case (and more above) is about a 1.1 model, so why do we need to set this flag which defaults to false anyway?

Reply:
I just want to explicit in all the flags that we set.